packages/web/server/airtable.ts

```diff
@@ -7,7 +7,8 @@ export default function airtableInit(baseID: string) {
   const { serverRuntimeConfig } = getConfig()
 
   if (!airTableSingleton) {
-    airTableSingleton = new AirtableAPI({ apiKey: serverRuntimeConfig.AIRTABLE_API_KEY })
+    // tslint:disable-next-line
+    airTableSingleton = new Airtable({ apiKey: serverRuntimeConfig.AIRTABLE_API_KEY })
```

Comment:
Are the types wrong here?

Reply:
ah these must have been left over from a trip down a different rabbit hole